src/java.base/share/native/launcher/main.c

```diff
@@ -34,6 +34,14 @@
 #include "jli_util.h"
 #include "jni.h"
 
+#ifdef UNDEFINED_BEHAVIOR_SANITIZER
+// Override weak symbol exposed by UBSan to override default options. This is called by UBSan
```

Comment:
Please explain why we need to override the default options.

Reply:
Done. Also added a comment regarding the environment variable UBSAN_OPTIONS.